db/schema.rb

```diff
@@ -23,7 +23,7 @@
 
   create_table "articles", id: :integer, charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
     t.string "title"
-    t.text "body", size: :medium
+    t.text "body", size: :long
```

Comment:
What's the source of this change (and similar ones below)?  I'm not seeing it in any migration or in current production.

Reply:
Resolving conversation.